includes/class-activitypub.php

```diff
@@ -332,18 +332,15 @@ public static function pre_get_avatar_data( $args, $id_or_email ) {
 			return $args;
 		}
 
+		/**
+		 * Filter allowed comment types for avatars.
+		 *
+		 * @param array $allowed_comment_types Array of allowed comment types.
+		 */
 		$allowed_comment_types = \apply_filters( 'get_avatar_comment_types', array( 'comment' ) );
-		if (
-			! empty( $id_or_email->comment_type ) &&
-			! \in_array(
-				$id_or_email->comment_type,
-				(array) $allowed_comment_types,
-				true
-			)
-		) {
+		if ( ! \in_array( $id_or_email->comment_type ?: 'comment', $allowed_comment_types, true ) ) { // phpcs:ignore Universal.Operators.DisallowShortTernary
```

Comment:
For context: Previously this conditional failed for an empty `comment_type`, which used to be the default for comments before WP 5.5. We no longer support WP versions that old, but I wonder if we should still just fallback to `comment`, instead of bailing and disabling avatars.

Reply:
happy to remove the fallback!